File: LeetCode/58. Length of Last Word/lengthOfLastWord.go
package main

import (
	"fmt"
	"strings"
)

func lengthOfLastWord(s string) int {
	listStr := strings.Fields(s)
	return len(listStr[len(listStr)-1])
}

func main() {
	fmt.Println(lengthOfLastWord("Hello World"))
	fmt.Println(lengthOfLastWord("   fly me   to   the moon  "))
	fmt.Println(lengthOfLastWord("luffy is still joyboy"))
}

File: LeetCode/58. Length of Last Word/lengthOfLastWord.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        listStr = s.split()
        return (len(listStr[-1]))

solution = Solution()
print(solution.lengthOfLastWord("Hello World"))
print(solution.lengthOfLastWord("   fly me   to   the moon  "))
print(solution.lengthOfLastWord("luffy is still joyboy"))